transform/mattermost-analytics/models/intermediate/product/servers/_int_servers__models.yml

```diff
@@ -34,6 +34,11 @@ models:
         description: The version of the database.
       - name: is_enterprise_ready
         description: Whether this server is an enterprise-ready build.
+      - name: binary_edition
```

Comment:
Do you think this will evolve into a more-encompassing `edition` field in the future or we'll have only `binary_edition` or keep the distinction between `binary_edition` and a future `edition`?

Reply:
IMHO that's one of the reasons we should use a string for enums (the other is to support the small number of `Unknown`). However, there doesn't seem to be a change on the editions for a long time. 

Another dimension vertical to this one is deployment type. I'd still keep them separated.